packages/salesforcedx-vscode-core/src/commands/forceOrgCreate.ts

```diff
@@ -192,7 +192,11 @@ const preconditionChecker = new CompositePreconditionChecker(
   new DevUsernameChecker()
 );
 const parameterGatherer = new CompositeParametersGatherer(
-  new FileSelector('config/**/*-scratch-def.json'),
+  new FileSelector(
```

Comment:
We create the scratch org definition files under the `config` folder because of the decisions made in the templates repo/plugin. However, the scratch org definition files can be anywhere in the project.
We should either allow the user to pick a file anywhere in the project (similar to how we pick where to create an apex class) or we should enforce the files to be under the `config` folder while also providing a command that creates a scratch org definition file for them.

cc @smaddox-sf

Reply:
I like that idea of creating a scratch def file for a user in the event that one is not present. We could also add some generic templates (like having one that inserts a namespace param). We could make some of the more common scenarios easier on customers. 

@smaddox-sf What are your thoughts on enforcing the config folder be the default and the creation of the scratch def file?

Reply:
@sfsholden, @lcampos  - i like the idea of creating a scratch def file for them so they don't have to go to documentation and then come back.  I think it makes sense to keep enforcing the config folder since the template repo needs that location.  It is a logical location, so i don't see an issue with that.

Reply:
Added W-7449375 to internal engineering for addressing this piece.